### DISCUSSION TITLE
Changing aircraft model in Unreal Engine

### DISCUSSION THREAD

**Discussion by Informatyk1234** [JSBSim-Team/jsbsim#1129] [Q&A]
How can I change visual aircraft model? I found only field to switch between Flight Dynamics Model,
![image](https://github.com/user-attachments/assets/23c9d034-17bf-4494-a8e8-b79ba4973b34)

 but I don't know how to connect it properly. Is there any tutorial how to do it?

**Comment by gallonmate** [comment]
Unreal Engine uses Skeletal Meshes and Static Meshes to render a 3D model. The plugin's BP_Airliner has a skeletal mesh to display the 3D aircraft model. For more information, you can search Unreal Engine tutorials regarding skeletal meshes. 
![image](https://github.com/user-attachments/assets/f73b432f-6850-4459-a665-927a51d2d0ef)
